docs/src/markdown-pages/aptitudes/user.createJWT.md

```diff
@@ -0,0 +1,5 @@
+---
+name: "JWT"
+links_js: "jwt"
+---
+Receive a JWT with the currently logged in Olive Helps user's email address in the Subject claim.
```

Comment:
I know the spec talked about other claims but I'm not sure whether they were all implemented - do we want to talk about what values are in each of them besides `exp`?